### PR TITLE
Make use content_for method

### PR DESCRIPTION
**Pull request by iguchi1124**
I'd prefer to use `content_for` tag when I want to set title tag from view partials.

This pull-request fixes to set title by using `content_for` and `yield` clause which are view helper methods.

20161230172101_img20161230-9-14xxy7t

### CODE REVIEW

**Comment by eagletmt** [comment]
Could you explain why you think `content_for` is preferable?

**Comment by iguchi1124** [comment]
Simple instance variable doesn't have informations enough, it is hard to know that where / how use to. that would be complicated.

The `content_for` method provides us explicitly way to yields shared html from view layer.

File: app/views/kuroko2/layouts/application.html.slim [hunk 1]
```diff
@@ -59,7 +59,7 @@ html
 
       aside.right-side
         section.content-header
-          h1= raw(@content_title || @title)
+          h1= raw yield(:content_title) || yield(:title)
```
Comment (hogelog) [inline app/views/kuroko2/layouts/application.html.slim]: `yield(:undefined_symbol)` returns `""` (empty string).

So `yield(:content_title) || yield(:title)` returns always `yield(:content_title)` value.
Do you want `yield(:content_title).presence || yield(:title)` here?

Comment (iguchi1124) [inline app/views/kuroko2/layouts/application.html.slim]: fixed

File: spec/features/dashborad_spec.rb [hunk 2]
```diff
@@ -20,6 +20,9 @@
       expect(page).to have_selector('#definitions_list table tbody tr', count: 1)
 
       expect(page).to have_content('There are no working jobs.')
+      expect(page).to have_title('Dashboard « Kuroko 2')
+      expect(page).to have_selector('i.fa.fa-dashboard', text: '')
+      expect(page).to have_content(' Dashboard')
```
Comment (eisuke) [inline spec/features/dashborad_spec.rb]: The ` Dashboard` may be displayed in other places.
How about using `have_selector('h1', text:  /Dashboard/)` ?

**Pull request merged**